proto/plan_common.proto

```diff
@@ -52,7 +52,8 @@ message ColumnCatalog {
 
 message CellBasedTableDesc {
   uint32 table_id = 1;
-  repeated OrderedColumnDesc order_key = 2;
+  repeated ColumnDesc columns = 2;
+  repeated OrderedColumnDesc order_key = 3;
```

Comment:
I think we should also store pk and distribution key here?

Reply:
You are very forward-looking! Plan to do this in next PRs. cc @st1page